spec/active_record_extensions_spec.rb

```diff
@@ -93,22 +91,19 @@ class Child < ActiveRecord::Base
 end
 
 describe ActiveRecordExtensions do
-  before(:all) do
-    Model = Class.new(ActiveRecord::Base) do
+  let(:model_class) {
+    Class.new(ActiveRecord::Base) do
       cache_all_attributes :by => 'name'
     end
-  end
-  after (:all) { Object.send(:remove_const, 'Model') }
+  }
 
   before do
+    connect_to_sqlite
+    stub_const 'Model', model_class
     allow(Model).to receive(:cache) { ActiveSupport::Cache::MemoryStore.new }
     allow(Model).to receive(:should_cache?) { true }
   end
 
-  after do
-    Model.delete_all
-  end
-
   it 'should cache all attributes' do
     @first = Model.create!(:name => 'First')
```

Comment:
I don't think these need to be instance variables, also, they should go in a before block within a context